libs/blocks/marketo/marketo.js

```diff
@@ -74,10 +96,23 @@ export const formSuccess = (form) => {
     return false;
   }
 
+  if (formData[SUCCESS_TYPE] === 'section') {
+    try {
+      const section = formData[SUCCESS_SECTION].toLowerCase().replaceAll(' ', '-');
+      const success = document.querySelector(`.section.${section}`);
+      success.classList.remove('hide-block');
+      success.scrollIntoView({ behavior: 'smooth' });
+      setPreference(SUCCESS_TYPE, 'message');
+    } catch (e) {
+      window.lana?.log('Error showing Marketo success section');
```

Comment:
Need to include tags for lana, for example:
```suggestion
      window.lana?.log('Error showing Marketo success section', { tags: 'errorType=warn,module=marketo' });
```

Reply:
Thanks!